modules/simpletest/resources/scripts/validationTest/securityTest.js

```diff
@@ -25,42 +27,42 @@ function doTest()
     });
 
     // LABKEY.Security.currentUser
-    if(!currentUser.isSystemAdmin)
+    if (!currentUser.isSystemAdmin)
         errors[errors.length] = Error("Security.currentUser = " + Ext.util.JSON.encode(currentUser));
 
     //LABKEY.Security.createContainer()
     var container = LABKEY.Security.createContainer({
         name: containerPath,
         containerPath: '/'
     });
-    if( container.path !== "/" + containerPath )
-        errors[errors.length] = new Error("Security.createContainer() = "+Ext.util.JSON.encode(container));
+    if (container.path !== "/" + containerPath)
+        errors[errors.length] = new Error("Security.createContainer() = " + Ext.util.JSON.encode(container));
 
     // LABKEY.Security.createGroup()
     var group = LABKEY.Security.createGroup({
         groupName: groupName,
         containerPath: containerPath
     });
-    if( group.name !== groupName )
-        errors[errors.length] = new Error("Security.createGroup() = "+Ext.util.JSON.encode(group));
+    if (group.name !== groupName)
+        errors[errors.length] = new Error("Security.createGroup() = " + Ext.util.JSON.encode(group));
 
     // LABKEY.Security.createNewUser()
     var user = LABKEY.Security.createNewUser({
         email: userEmail,
         sendEmail: false,
         containerPath: containerPath
     });
-    if( user.email !== userEmail )
-        errors[errors.length] = new Error("Security.createNewUser() = "+Ext.util.JSON.encode(user));
+    if (user.email !== userEmail)
+        errors[errors.length] = new Error("Security.createNewUser() = " + Ext.util.JSON.encode(user));
 
     // LABKEY.Security.addGroupMembers()
     result = LABKEY.Security.addGroupMembers({
         groupId: group.id,
         principalIds: [user.userId],
         containerPath: containerPath
     });
-    if(result.added != user.userId )
-        errors[errors.length] = new Error("Security.addGroupMembers() = "+Ext.util.JSON.encode(result));
+    if (result.added[0] !== user.userId)
```

Comment:
Should this guard against the (remote) possibility of `result.added` not having a single item?
```suggestion
    if (result.added.length !== 1 || result.added[0] !== user.userId)
```

Reply:
Yes!